images/ovn-kubernetes.yml

```diff
@@ -13,15 +14,15 @@ enabled_repos:
 from:
   builder:
   - stream: golang
-  stream: rhel
+  member: openshift-enterprise-base
 labels:
   License: GPLv2+
   io.k8s.description: This is a component of OpenShift Container Platform that provides
     an overlay network using ovn.
   io.k8s.display-name: ovn kubernetes
   io.openshift.tags: openshift
   vendor: Red Hat
-name: openshift/ose-ovn-kubernetes
+name: openshift/ovn-kubernetes
```

Comment:
Likewise for the image name. Since it's non-release I guess the ose- is optional but is there any particular reason to change it?